app/views/campaigns/index.html.haml

```diff
@@ -27,3 +27,19 @@
           = link_to t('.edit'), edit_campaign_path(campaign), class: 'btn btn-outline-info btn-campaigns'
           = link_to t('.delete'), campaign_path(campaign), method: :delete,
             data: { confirm: t('.are_you_sure') }, class: 'btn btn-outline-info btn-campaigns'
+        .style-dropdown
+          %span= t('.campaign_state_is')
+          %span.dropdown
+            %button.btn.btn-outline-info.dropdown-toggle(data-toggle='dropdown' type='button')
+              %span= campaign.aasm_state
+            .dropdown-menu
+              - if campaign.aasm_state == 'new'
```

Comment:
I guess there should be some helper methods like `campaign.new?` `campaign.published?` etc